### DISCUSSION TITLE
A new build

### DISCUSSION THREAD

**Discussion by openenglishbible** [openenglishbible/Open-English-Bible#347] [Announcements]
Just as an update, I've started the process for building a release, updating the bitrot on the generation software etc.
Cheers Russell

**Comment by openenglishbible** [comment]
OK, I've pushed the update as 2022.1 and updated the website.  I'll do another post shortly setting out a plan for the next step...

**Reply by RetroGamer02** [reply]
Hi I noticed that some of the links for downloading the new version don't work. Such as the plain text version.

**Reply by russellallen** [reply]
Hi - I've tried to fix the issue. Let me know if it still isn't working for you!

**Reply by RetroGamer02** [reply]
It works now ty. =)
